### PR TITLE
clear-all now at the bottom, merge first, then logo, then preview hierarchy

### PR DESCRIPTION
**Pull request by siteations**
does not include full pattern fixes or the final font stuff, fyi.

David- I'd confirm the clear-all is the behavior we want.... it's just setting boxes back to the initial box. Does it need to do anything else to keep the store happy?

### CODE REVIEW

File: src/reducers/boxes.js [hunk 1]
```diff
@@ -53,6 +53,8 @@ const boxesReducer = (prevState = initialState, action) => {
       return prevState.clear().merge(action.newLayout);
     case SAVE:
       return prevState;
+    case CLEAR_ALL:
```
Comment (thenorthstarblues) [inline src/reducers/boxes.js:56]: have you tested this? might need to be return prevState.clear().mergeWith(initialState)

**Review by thenorthstarblues — CHANGES_REQUESTED** [review]

File: src/reducers/boxes.js (hunk 1 above)
Comment (siteations) [reply src/reducers/boxes.js:56]: will write that version and see what the state difference looks like. thanks!

**Comment by siteations** [comment]
so the store- in redux preview - seems the same either way during testing. It seems better to be consistent with formatting. The other question is whether this resetting to the initial immutable will fuck with the undo/redo, if a user decides they want to undo a clear.

**Comment by thenorthstarblues** [comment]
Undo probably won't end up working anyway because it will go one step at a time...since we are updating the store every time that they adjust boxes even slightly, it would take the user back each tiny step at a time, so probably not too useful. Does clear all need to be a component? Seems like we could save the code base more clutter by just making it a button

**Comment by siteations** [comment]
it could be a button if we carve out some space or weed out other div addition options... maybe at the bottom of the fontOptions bar.

**Comment by siteations** [comment]
which is to say - i'll shift that out tonight when I work on the font bar... carry on.

**Review by thenorthstarblues — APPROVED** [review]

**Pull request merged**